### PR TITLE
Http4s implementation

### PR DESCRIPTION
**Pull request by jendakol**

### CODE REVIEW

File: http4s/src/main/scala/com/avast/grpc/jsonbrige/http4s/Http4s.scala [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+import scala.concurrent.ExecutionContext
+
+object Http4s {
+  def apply(configuration: Configuration)(bridges: GrpcJsonBridge[_ <: BindableService]*)(
+      implicit ec: ExecutionContext): HttpService[IO] = {
+    val services = bridges.map(s => (s.serviceName, s): (String, GrpcJsonBridge[_])).toMap
+
+    HttpService[IO] {
+      case request @ POST -> Root / serviceName / methodName =>
```
Comment (petr-k) [inline http4s/src/main/scala/com/avast/grpc/jsonbrige/http4s/Http4s.scala]: What about the route listing available operations? Did we decide to drop it?

Comment (petr-k) [inline http4s/src/main/scala/com/avast/grpc/jsonbrige/http4s/Http4s.scala]: I imagine I could want the "base path" configurable, especially when bridging gRPC calls alongside having other app-specific routes. Maybe adding a parameter to the config, with `Root` being the default?

File: http4s/src/test/scala/com/avast/grpc/jsonbrige/http4s/Http4sTest.scala [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+      .apply(
+        Request[IO](method = Method.POST,
+                    uri = Uri.fromString(s"${classOf[TestApiServiceImplBase].getName.replace("$", ".")}/Get").getOrElse(fail()))
+          .withBody(""" { "names": ["abc","def"] } """)
+          .unsafeRunSync()
+      )
+      .value
+      .unsafeRunSync()
+
+    assertResult(org.http4s.Status.Ok)(response.status)
```
[the remark this answers is not in the document]
Comment (jendakol) [reply http4s/src/test/scala/com/avast/grpc/jsonbrige/http4s/Http4sTest.scala]: Done

**Comment by jendakol** [comment]
@petr-k Updated and fixed.

**Pull request merged**